src/utils/time.test.js

```diff
@@ -48,4 +55,89 @@ describe('time', () => {
       expect(formatDateTime(datetime)).toEqual('20 Jan 2021, 12:16 am')
     })
   })
+
+  describe('beginningOfDay', () => {
+    it('sets the time to the beginning of the day', () => {
+      const date = new Date('2021-02-17T15:16:57Z')
+      expect(beginningOfDay(date)).toEqual(new Date('2021-02-17T00:00:00Z'))
+    })
+  })
+
+  describe('beginningOfWeek', () => {
+    describe('when the day is Monday', () => {
+      it('sets the date/time to the beginning of the week', () => {
+        const date = new Date('2021-02-15T15:16:57Z')
+        expect(beginningOfWeek(date)).toEqual(new Date('2021-02-15T00:00:00Z'))
+      })
+    })
+
+    describe('when the day is Tuesday', () => {
+      it('sets the date/time to the beginning of the week', () => {
+        const date = new Date('2021-02-16T15:16:57Z')
+        expect(beginningOfWeek(date)).toEqual(new Date('2021-02-15T00:00:00Z'))
+      })
+    })
+
+    describe('when the day is Wednesday', () => {
+      it('sets the date/time to the beginning of the week', () => {
+        const date = new Date('2021-02-17T15:16:57Z')
+        expect(beginningOfWeek(date)).toEqual(new Date('2021-02-15T00:00:00Z'))
+      })
+    })
+
+    describe('when the day is Thursday', () => {
+      it('sets the date/time to the beginning of the week', () => {
+        const date = new Date('2021-02-18T15:16:57Z')
+        expect(beginningOfWeek(date)).toEqual(new Date('2021-02-15T00:00:00Z'))
+      })
+    })
+
+    describe('when the day is Friday', () => {
+      it('sets the date/time to the beginning of the week', () => {
+        const date = new Date('2021-02-19T15:16:57Z')
+        expect(beginningOfWeek(date)).toEqual(new Date('2021-02-15T00:00:00Z'))
+      })
+    })
+
+    describe('when the day is Saturday', () => {
+      it('sets the date/time to the beginning of the week', () => {
+        const date = new Date('2021-02-20T15:16:57Z')
+        expect(beginningOfWeek(date)).toEqual(new Date('2021-02-15T00:00:00Z'))
+      })
+    })
+
+    describe('when the day is Sunday', () => {
+      it('sets the date/time to the beginning of the week', () => {
+        const date = new Date('2021-02-21T15:16:57Z')
+        expect(beginningOfWeek(date)).toEqual(new Date('2021-02-15T00:00:00Z'))
+      })
+    })
+  })
+
+  describe('daysAfter', () => {
+    it('sets the time to the beginning of the day', () => {
```

Comment:
I guess this could be `sets the time to the supplied number of days in the future` or similar.